ouroboros-consensus-diffusion/src/Ouroboros/Consensus/Node.hs

```diff
@@ -55,8 +55,8 @@ import qualified Codec.CBOR.Decoding as CBOR
 import qualified Codec.CBOR.Encoding as CBOR
 import           Codec.Serialise (DeserialiseFailure)
 import           Control.Monad.Class.MonadMVar (MonadMVar)
-import           Control.Monad.Class.MonadTime (MonadTime)
-import           Control.Monad.Class.MonadTimer (MonadTimer)
+import           Control.Monad.Class.MonadTime.SI (MonadTime)
```

Comment:
I wonder if it'd make sense to re-export these symbols in some `io-sim` module to avoid this amount of churn. Not for this PR of course, but maybe something to consider for the future.

Reply:
`Control.Monad.Class.MonadTime.SI` comes from `si-timers` not even `io-classes`.  We'd need to have some kind of `io-classes-prelude` package for that.